test/evergreen.yml

```diff
@@ -961,6 +979,16 @@ tasks:
           CLANG_C_VERSION: -DCLANG_C_VERSION=8
           CLANG_CXX_VERSION: -DCLANG_CXX_VERSION=8
 
+  # Base compile for nonstandalone build
+  - name: compile-nonstandalone
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          NONSTANDALONE: -DWT_STANDALONE_BUILD=0
```

Comment:
It feels worthwhile to retrieve this section of code into a separate function, as it's called multiple times in the file.

Reply:
It is already a function. Not sure how would you do it.